action.yml

```diff
@@ -0,0 +1,22 @@
+name: composite-action-lint
+
+description: actionlint for composite actions
+
+inputs:
+  actions:
+    description: space separated list of paths to actions to lint, supports wildcards
+    default: .github/actions/*/action.yml
```

Comment:
I would be interested to know if you ever saw this convention of putting composite actions in `.github/actions/` outside of bettermarks?
I'm not sure specifically about composite actions, but my expectation would be that the most common metadata file location would be - like in this repo - the top level `action.yml`.

...
I very quickly found an example at https://github.com/dagger/dagger/tree/main/.github/actions ...
I think I will likely never submit this review 🙃